bin/view_godocs

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Moving out the repo into `src` etc. etc. is a bit of a pain even with Symlinks. Since the project is already locked into using Docker for development I think the following might be a cleaner experience with no need for additional instructions: 

```
#!/bin/bash

# This is a convenience script to start go doc for the secretless-repo

echo "godoc is starting up..."
echo ""
echo "The public Secretless docs will be viewable at:"
echo ""
echo "    http://localhost:6060/pkg/secretless-broker/"
echo ""

docker run \
  --rm \
  -it \
  -p 6060:6060 \
  -v $PWD:/go/src/secretless-broker \
  -w /go/src/secretless-broker \
  golang:1.12.5-stretch \
    godoc -http=:6060

```